pep-0387.txt

```diff
@@ -48,31 +49,42 @@ This policy applies to all public APIs.  These include:
 - Behavior of classes with regards to subclasses: the conditions under
   which overridden methods are called.
 
+- Documented exceptions and the semantics which lead to their raising.
+
+- Exceptions commonly raised in EAFP scenarios.
+
 Others are explicitly not part of the public API.  They can change or
 be removed at any time in any way.  These include:
 
-- Function, class, module, attribute, method, and C-API names and types that
-  are prefixed by "_" (except special names).  The contents of these
-  are also not subject to the policy.
+- Function, class, module, attribute, method, and C-API names and
+  types that are prefixed by "_" (except special names).
+
+- Anything documented publicly as being private.
+
+- Imported modules (unless explicitly documented as part of the public
```

Comment:
Does this mean, for example, `import lzma; lzma.io.open()` isn't covered?

Reply:
Correct. Unless it's documented that `lzma.io` is a thing, it isn't backwards incompatible to remove it.

Reply:
Okay. It might be clearer to say "modules imported as a dependencies" or something.